teacher.rb

```diff
@@ -1,8 +1,15 @@
 require_relative 'school_member'
-require_relative 'payable'
 
 class Teacher < SchoolMember
-	include Payable
+
+	def salary=(new_salary)
```

Comment:
@salogel42 Not sure why the indenting looks weird here, since it looks ok on Sublime.